product_brand_tag/models/product_brand_tag.py

```diff
@@ -2,10 +2,12 @@
 # @author: Simone Orsi <simone.orsi@camptocamp.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import models
+from odoo import fields, models
 
 
 class ProductBrandTag(models.Model):
     _name = "product.brand.tag"
     _inherit = "product.brand.tag.mixin"
     _description = "Product Brand Tag"
+
+    product_brand_ids = fields.Many2many(relation="product_brand_tag_rel")
```

Comment:
I did it in extending modules only https://github.com/OCA/brand/blob/14.0/product_brand_social_responsibility/models/product_brand_tag_csr.py but definitely better to do it here as well :+1: